website/src/pages/index.tsx

```diff
@@ -0,0 +1,126 @@
+import React from "react";
+import Link from "@docusaurus/Link";
+import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
+import Layout from "@theme/Layout";
+import * as icons from "../icons";
+
+import styles from "./index.module.css";
+
+type FeatureItem = {
+  title: string;
+  Icon: React.ComponentType<React.SVGProps<SVGSVGElement>>;
+  description: string;
+};
+
+const FeatureList: FeatureItem[] = [
+  {
+    title: "Pub/Sub logging",
+    Icon: icons.Robot,
+    description:
+      "MCAP is ideal for storing multiple channels of timestamped log data, such as pub/sub messages or multimodal sensor data.",
+  },
+
+  {
+    title: "Serialization agnostic",
+    Icon: icons.DrawerEnvelope,
+    description:
+      "MCAP is a container format, allowing you to record and replay binary messages in any format, such as Protobuf, DDS (CDR), ROS, JSON, etc.",
+  },
+  {
+    title: "High performance writing",
+    Icon: icons.SportsCarConvertible,
+    description:
+      "MCAP utilizes a row-oriented, append-only design. This minimizes disk I/O, and reduces the risk of data corruption during an unclean shutdown. ",
+  },
+  {
+    title: "Self-contained",
+    Icon: icons.ShipmentPackage,
+    description:
+      "MCAP files are fully self-contained, including schemas required to deserialize each channel. Older files always remain readable, even as your codebase evolves.",
+  },
+  {
+    title: "Efficient seeking",
+    Icon: icons.Turntable,
+    description:
+      "MCAP files contain an optional index, allowing for fast, efficient reading, even over a low-bandwidth internet connection.",
+  },
+  {
+    title: "Optional compression",
+    Icon: icons.ZipFile,
+    description:
+      "Data can be compressed using lz4 or zstd, while still supporting efficient indexed reads and chunk-based decompression.",
+  },
+  {
+    title: "Multilingual",
+    Icon: icons.ChatTranslate,
+    description:
+      "MCAP libraries are available in many languages, including C++, Go, Python, Rust, Swift, Typescript.",
+  },
+  {
+    title: "Flexible",
+    Icon: icons.YogaLegGrabStretch,
+    description:
+      "Features such as chunking, indexing, CRC checksums, and compression are optional. You choose the right tradeoffs for your application.",
+  },
+  {
+    title: "Production grade",
```

Comment:
```suggestion
    title: "Production-grade",
```

Reply:
why hyphenate? these seem like separate words

Reply:
https://www.grammarly.com/blog/hyphen-with-compound-modifiers/

Reply:
Ok I accepted your other suggestion, but I still disagree with this one because it is not clearly an adjective in a sentence.

E.g. `production-grade software` seems correct to hyphenate but `mcap is production grade` should not have a hyphen.

https://www.stcloudstate.edu/writeplace/_files/documents/grammar-punctuation/hyphen-rules.pdf

Reply:
yeah I guess I applied the rule arbitrarily in my head because it's a sentence fragment.

On the other hand, I don't know actually know if the rule applies to things like this where the second word of the pair is a noun. For example, `top-tier` is just a word in the dictionary: https://www.merriam-webster.com/dictionary/top-tier It's not acting like two separate words that happen to be hyphenated 🤷   In my mind `production-grade` should follow the same rule as `top-tier` but I can't really back that up with sources.